### ISSUE TITLE
[Feature request] Turn off profiles 

### ISSUE THREAD

**Issue by Ircas** [streamaserver/streama#820]
I understand that netflix is the best way to go as from where you should get inspiration but I would like streama to have more options in the settings menu. 

I would like to see a button to enable disable some feutures as 

Profile selection and creation 
Subtitles /audio
Video sychronization/share
Login/Mentainance mode (if enabled only admins should enter) 

Of course my first priotity is profiles (thus in the title). Some ppl that have not used netflix before could find them a bit confusing.

My setup is for friends and family that cannot or dont want to pay a monthly fee. 
They range from 15 to 75 years old. 

I would like to have a movie tv show and video library as easy,clan and minimalistic as possible especially for the elder ones. 

I hope someone likes this and wants those feutures too. 

The more people asking for the same stuff the larger the possibility those will actually make it in development.

**Comment by jkoop** [comment]
I agree. And I'll add to the list a bit: Turn off "Recommendations for You" in User Settings
